src/intree.js

```diff
@@ -114,6 +123,11 @@ module.exports.setup = function(cfg) {
     if (config.tasks.length > 0) {
       let taskGroupId = config.tasks[0].taskId;
       config.tasks = config.tasks.map((task) => {
+        // Temporary for debugging purposes
+        if (task.task.extra.github.events.some(event => event == 'release')) {
```

Comment:
Well, it comes from [here](https://github.com/taskcluster/taskcluster-github/blob/master/src/intree.js#L101-L104 ) ... I think it should be correct.